packages/create-flex-plugin/templates/js/src/DemoPlugin.js

```diff
@@ -13,15 +16,31 @@ export default class {{pluginClassName}} extends FlexPlugin {
    * This code is run when your plugin is being started
    * Use this to modify any UI components or attach to the actions framework
    *
-   * @param flex { typeof import('@twilio/flex-ui') }
-   * @param manager { import('@twilio/flex-ui').Manager }
+   * @param flex { typeof Flex }
+   * @param manager { Flex.Manager }
```

Comment:
The original comments were quite useful to get Intellisense support for Flex even without Typescript.
From: https://github.com/Microsoft/vscode/issues/53093